storagemarket/types/deal_state.go

```diff
@@ -39,6 +39,8 @@ type ProviderDealState struct {
 	ChainDealID abi.DealID
 	PublishCID  *cid.Cid
 
+	IndexerAnnouncementCID *cid.Cid
```

Comment:
Why do we need the `IndexerAnnouncementCID` on the deal state?

Reply:
Actually, just logging it should be good enough for debugging as we will have it in the deal log. Removing this.

Reply:
This is done.